### ISSUE TITLE
External users in circles

### ISSUE THREAD

**Issue by thammerbacher** [nextcloud/circles#153]
You can only add local users and groups to circles. It would be kind to create a function that allows to add external users (identified by ldap or sso or whatever) to circles!

**Comment by sunjam** [comment]
Not sure if a different issue, but I'm looking to add federated users to circles.

**Comment by mokkin** [comment]
Meanwhile it is possible, to add contacts (emails) to a circle, that are no users.
But what happens to them? They are not getting a guest account as I can see.

**Comment by hirunatan** [comment]
(bump) I've come to this situation and see it's the same: I can add an external user to a circle, but they don't receive any invitation or any means to access the nextcloud instance.

**Comment by Uatschitchun** [comment]
> (bump) I've come to this situation and see it's the same: I can add an external user to a circle, but they don't receive any invitation or any means to access the nextcloud instance.

Same

**Comment by wunschadresse** [comment]
I do not think it is meant to invite people to the platform. It is simple a means of keeping track of people (and contacting via mail) for people who are not supposed to be or do not want to be on the platform. same issue her: https://github.com/nextcloud/circles/issues/1248
